Science/.gitignore

```diff
@@ -1,12 +0,0 @@
-**/.DS_Store
```

Comment:
This .gitignore doesn't have anything that isn't already ignored by the top-level .gitignore.

Reply:
It doesn't hurt to have something ignored more than once. It's actually a good idea to have a .gitignore everywhere you have a .sln. This file should be readded